zoo-css-challenge/style.css

```diff
@@ -189,7 +240,9 @@ footer {
 }
 
 /*Programs*/
-
+.cards {
+  display: flex;
```

Comment:
This looks better, but the cards could be spread out evenly across the page. The model answer for this suggests something like 
'width: calc(calc(100% / 3) - 65px);` is present on .card and `.programs { display: block } has been removed` OR flexbox or grid is present'

Reply:
Thanks. I've remove {display: block} and present .cards {display: flex; justify-content: space-evenly} Is this acceptable?